app/controllers/org_admin/templates_controller.rb

```diff
@@ -373,5 +318,10 @@ def template_options()
     def plan_params
       params.require(:plan).permit(:org_id, :funder_id)
     end
+    
+    def template_type(template)
```

Comment:
I think this method should leave in a helper for the template not here and since it is used only in the controller you can include the module instead of using the helper keyword from rails